### PR TITLE
:bug: Fix the integration test

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

## Related issue(s)

Fixes #

### CODE REVIEW

File: test/integration/operator/hubofhubs/transporter_test.go [hunk 1]
```diff
@@ -73,19 +73,22 @@ var _ = Describe("transporter", Ordered, func() {
...
 		reconciler := operatortrans.NewTransportReconciler(runtimeManager)
 
-		err = reconciler.Reconcile(ctx, mgh)
-		Expect(err).To(Succeed())
-
-		// the connection is generated
-		conn = config.GetTransporterConn()
-		Expect(conn).NotTo(BeNil())
+		Eventually(func() error {
+			err = reconciler.Reconcile(ctx, mgh)
```
Comment (clyang82) [inline test/integration/operator/hubofhubs/transporter_test.go:79]: eventually reconciling?

**Comment by yanmxa** [comment]
/test test-integration

run the test to verify its stability!

**Comment by yanmxa** [comment]
run the integration test three times and it's all successful. 

/cc @clyang82

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**